include/sys/zfs_context.h

```diff
@@ -154,53 +154,6 @@ extern void vpanic(const char *, va_list)  __NORETURN;
 
 extern int aok;
 
-/*
- * DTrace SDT probes have different signatures in userland than they do in
- * the kernel.  If they're being used in kernel code, re-define them out of
- * existence for their counterparts in libzpool.
- *
- * Here's an example of how to use the set-error probes in userland:
- * zfs$target:::set-error /arg0 == EBUSY/ {stack();}
- *
- * Here's an example of how to use DTRACE_PROBE probes in userland:
- * If there is a probe declared as follows:
- * DTRACE_PROBE2(zfs__probe_name, uint64_t, blkid, dnode_t *, dn);
- * Then you can use it as follows:
- * zfs$target:::probe2 /copyinstr(arg0) == "zfs__probe_name"/
- *     {printf("%u %p\n", arg1, arg2);}
- */
-
-#ifdef DTRACE_PROBE
-#undef	DTRACE_PROBE
-#endif	/* DTRACE_PROBE */
-#define	DTRACE_PROBE(a) \
-	ZFS_PROBE0(#a)
-
-#ifdef DTRACE_PROBE1
-#undef	DTRACE_PROBE1
-#endif	/* DTRACE_PROBE1 */
-#define	DTRACE_PROBE1(a, b, c) \
-	ZFS_PROBE1(#a, (unsigned long)c)
-
-#ifdef DTRACE_PROBE2
-#undef	DTRACE_PROBE2
-#endif	/* DTRACE_PROBE2 */
-#define	DTRACE_PROBE2(a, b, c, d, e) \
-	ZFS_PROBE2(#a, (unsigned long)c, (unsigned long)e)
-
-#ifdef DTRACE_PROBE3
-#undef	DTRACE_PROBE3
-#endif	/* DTRACE_PROBE3 */
-#define	DTRACE_PROBE3(a, b, c, d, e, f, g) \
-	ZFS_PROBE3(#a, (unsigned long)c, (unsigned long)e, (unsigned long)g)
-
-#ifdef DTRACE_PROBE4
-#undef	DTRACE_PROBE4
-#endif	/* DTRACE_PROBE4 */
-#define	DTRACE_PROBE4(a, b, c, d, e, f, g, h, i) \
-	ZFS_PROBE4(#a, (unsigned long)c, (unsigned long)e, (unsigned long)g, \
-	(unsigned long)i)
-
```

Comment:
Unless there's a real need to move this, I'd suggest we leave all of this in the user space portion of the `zfs_context.h` for now.